src/main/kotlin/com/hypto/iam/server/validators/RequestModelValidator.kt

```diff
@@ -449,6 +450,17 @@ val updateUserRequestValidation = Validation {
     }
 }
 
+val createSubOrganizationRequest = Validation {
+    CreateSubOrganizationRequest::name required {
+        maxLength(MAX_SUB_ORG_LENGTH) hint "Maximum length supported for" +
+            "name is $MAX_SUB_ORG_LENGTH characters"
+        noEndSpaces()
```

Comment:
Will this validate or mutate the input string?

Reply:
noEndSpaces() should not be used. Updated in this PR https://github.com/hwslabs/iam/pull/163